Formula/g/gstreamer.rb

```diff
@@ -249,6 +249,13 @@ def post_install
     libnice_gst_plugin = Formula["libnice-gstreamer"].opt_libexec/"gstreamer-1.0"/shared_library("libgstnice")
     gst_plugin_dir = lib/"gstreamer-1.0"
     ln_sf libnice_gst_plugin.relative_path_from(gst_plugin_dir), gst_plugin_dir
+
+    # The initial plugin load takes a long time without extra permissions on
+    # macOS, which frequently causes the slower Intel macOS runners to time
+    # out during test. Therefore we generate the cache during postinstall.
+    #
+    # Ref: https://gitlab.freedesktop.org/gstreamer/gstreamer/-/issues/1119
+    quiet_system bin/"gst-inspect-1.0" if OS.mac? && Hardware::CPU.intel? && ENV["HOMEBREW_GITHUB_ACTIONS"]
```

Comment:
Though this will slow down testing formula that have `gstreamer` as dependency. Not sure if there is an ideal option or we just keep skipping the test.

Reply:
Actually, this won't work as it will set up cache in user directory rather than `HOME`-specificed test directory